### PR TITLE
add linux mint 19 support

### PR DESCRIPTION
**Pull request by dr3dd589**
Linux Mint is based on Debian and Ubuntu so all are same only os name and version is changed.

Please submit your pull requests to the 'dev' branch.

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant (Dojo's code isn't currently flake8 compliant, but we're trying to correct that)
- [ ] If this is a new feature and not a bug fix, you've included the proper documentation under the /docs folder

### CODE REVIEW

File: entrypoint_scripts/common/common-os.sh [hunk 1]
```diff
@@ -115,6 +115,15 @@ bootstrap_install() {
 	    echo "Bootstapping CentOS"
 	    echo "  TBD: Pre-reqs for CentOS"
 	    ;;
+	    "Linux Mint")
```
Comment (ppiotr3k) [inline entrypoint_scripts/common/common-os.sh]: As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
	    "Ubuntu" | "Linux Mint")
```

File: entrypoint_scripts/os/linux.sh [hunk 2]
```diff
@@ -297,6 +301,11 @@ function install_linux() {
 		    echo "  Installing database on CentOS"
 		    echo "  TBD: DB install for CentOS"
 		    ;;
+		    "Linux Mint")
```
Comment (ppiotr3k) [inline entrypoint_scripts/os/linux.sh]: As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
		    "Ubuntu" | "Linux Mint")
```

File: entrypoint_scripts/os/linux.sh [hunk 3]
```diff
@@ -320,6 +329,14 @@ function install_linux() {
         echo "  Installing database on CentOS"
 		echo "  TBD: DB install for CentOS"
 		;;
+	    "Linux Mint")
```
Comment (ppiotr3k) [inline entrypoint_scripts/os/linux.sh]: As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
	    "Ubuntu" | "Linux Mint")
```

File: entrypoint_scripts/os/linux.sh [hunk 4]
```diff
@@ -97,6 +97,10 @@ function ubuntu_wkhtml_install() {
 	    wget https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.xenial_amd64.deb
 	    dpkg -i wkhtmltox_0.12.5-1.xenial_amd64.deb 
 	    ;;
+        "19")
```
Comment (ppiotr3k) [inline entrypoint_scripts/os/linux.sh]: As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
        "18.04" | "19")
```

**Comment by dr3dd589** [comment]
@ppiotr3k done! Can you please review it.

**Comment by ppiotr3k** [comment]
> @ppiotr3k done! Can you please review it.

Looks much better! Thank you :smiley:

**Pull request merged**